text/0000-sigstore-integration.md

```diff
@@ -0,0 +1,387 @@
+- Feature Name: cargo_crates_sigstore_integration
+- Start Date: 2023-03-27
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+Authors:
+
+* Tim Pletcher, Hewlett Packard Enterprise
+* Ulf Lilleengen, Red Hat / IBM
+
+# Summary
+[summary]: #summary
+
+The Solar Winds breach event was an inflection point for the topic of software supply chain security. The Solar Winds event exposed weaknesses in two specific areas, artifact provenance and build system integrity. That event resulted in a broad variety of reactions ranging from an aggressive push by the USG to a rapid acceleration of security/provenance related OSS and commercial projects oriented on addressing the significant shortcomings in the OSS software ecosystem.
```

Comment:
I feel like this document is a lot more... corporate-oriented than is usually seen in Rust.  To be sure, lots of people working on Rust and Rust crates do so in the scope of employment by a large corporation, but I suspect some readers will have a reaction like, "why should I give a hoot what the US government wants?".  And they may recall reading blog posts like [this one](https://iliana.fyi/blog/software-supply-chain/) and [this one](https://lucumr.pocoo.org/2022/7/9/congratulations/) that take issue with the entire concept of supply chain security.

Admittedly, the specific policies being complained about in this posts are largely orthogonal to this RFC which is just about signing crates.  And in fact I think the proposed approach makes a lot of sense.  But I think the wording could do a better job of *selling it*.

Why do I want this?  Not because the White House is pushing for it.  But because I don't want malware on my computer or in the software I publish, and the proposed approach (theoretically) makes signing so easy that there's no reason not to do it.

Reply:
Thanks for the feedback! We will reduce the corp-speak and size of the summary, my apologies for that. I think the reason it ended up this way is that we were not sure how much background context we had to provide for this kind of RFC. 

I agree, probably lots of crate maintainers don't care about what the US gov wants, I consider it more a reference of how important this issue is to resolve for open source Rust software to be trustable longer term.